homeassistant/components/fan/icons.json

```diff
@@ -23,6 +23,7 @@
     "decrease_speed": "mdi:fan-minus",
     "increase_speed": "mdi:fan-plus",
     "oscillate": "mdi:arrow-oscillating",
+    "set_direction": "mdi:rotate-right",
```

Comment:
I think that `rotate-right` would be better here

Reply:
```suggestion
    "set_direction": "mdi:rotate-3d-variant",
```